runner/webserver.ts

```diff
@@ -142,12 +143,13 @@ export function webserver(wct: Context): void {
     // webserver as they please.
     await wct.emitHook('prepare:webserver', app);
 
-    // Serve up all the static assets.
-    app.use(serveWaterfall(wsOptions.pathMappings, {
+    // Serve up project & dependencies via polyserve
+    const polyserve = makeApp({
       root: options.root,
       headers: DEFAULT_HEADERS,
-      log: wct.emit.bind(wct, 'log:debug'),
-    }));
+      packageName: path.basename(options.root),
```

Comment:
That's good to know about the default behavior. I'd rather keep this matching current wct behavior as much as possible and not add a dependency to bower for now, i'll remove the option and let polyserve use it's default.